src/Util/ReflectionClassRecursiveIterator.php

```diff
@@ -45,7 +45,12 @@ public static function getReflectionClassesFromDirectories(array $directories):
                     $sourceFile = realpath($sourceFile);
                 }
 
-                require_once $sourceFile;
+                try {
+                    require_once $sourceFile;
+                } catch (\Throwable $t) {
+                    // invalid PHP file (example: missing parent class)
```

Comment:
should'nt we log at least so that the user knows there's an issue in this class?

Reply:
It's usually not an issue but an optional dependency not installed or something like that. If the class is used elsewhere, PHP will throw an error anyway.